api/src/data_inclusion/api/inclusion_data/commands.py

```diff
@@ -144,8 +144,12 @@ def load_inclusion_data():
             )
         ]
 
-    structure_data_list = structures_df.to_dict(orient="records")
-    service_data_list = services_df.to_dict(orient="records")
+    structure_data_list = structures_df.sort_values(
+        by="_di_surrogate_id", ascending=True
+    ).to_dict(orient="records")
+    service_data_list = services_df.sort_values(
+        by="_di_surrogate_id", ascending=True
+    ).to_dict(orient="records")
```

Comment:
Soit ça, soit (comme le _di_surrogate_id a déjà un index)  on peut s'assurer que l'on fait des where/order by sur des colonnes indexées ?

Reply:
les 2 colonnes sont déjà indexées

Reply:
Oui oui c'est ce que je dis !
- si elles sont déjà indexées (c'est la cas)
- ... ET que les clauses WHERE et ORDER BY qui suivent se font sur des colonnes indexées (not sure)
- ... alors ce tri n'est pas forcément nécessaire ? il peut etre remplacé par les index. Après c'est peut etre moins pénible ainsi.